src/pages/guides/review/index.md

```diff
@@ -76,20 +153,17 @@ Additionally, your integration must not violate the trademarks and copyrights of
 * If the integration lets users start with an existing asset, the iFrame should load with the user-selected image.
 * The export buttons appear as the developer declared (in the "Notes for reviewers" section) during submission. Custom buttons should behave as defined.
 
-### Common rejection reasons
+## Common rejection reasons
 
-1. Your app or integration is not accessible via free user sign-up and test credentials are not provided to our review team for testing purposes.
-2. Your app description is not complete in the __Notes to reviewer__ section, and our review team cannot test all the entrypoints. Make sure you are clear about what your app integrates (full editor, modules, and/or quick actions) and where/how to invoke them.
-3. Your app name should correspond with the name of your project in Adobe Developer Console. This `appName` should also be consistent throughout your application. We use this property in our analytics checks so if there are any discrepancies, we will ask that you make that change before you are able to release the integration to your users.
-
-## After you submit
-
-Congratulations! You're on your way to getting your integration into the hands of your users. Our teams are working on new submissions every day. Depending on volume, it can be up to 10 business days before you hear a response from us. On average, we approve integrations within 3 business days. Thank you for your patience as we review your submission.
-
-Once your submission has been reviewed, we will email you letting you know whether your submission was approved or rejected. If it was rejected, we will include the reasons for rejection along with any next steps. If you'd like to back and edit your submission, you can do so in the Developer Console.
+-  Your app or integration is not accessible via free user sign-up and test credentials are not provided to our review team for testing purposes.
+- Your app description is not complete in the __Notes to reviewer__ section, and our review team cannot test all the entrypoints. Make sure you are clear about what your app integrates (full editor, modules, and/or quick actions) and where/how to invoke them.
+- Your app name should correspond with the name of your project in Adobe Developer Console. This `appName` should also be consistent throughout your application. We use this property in our analytics checks so if there are any discrepancies, we will ask that you make that change before you are able to release the integration to your users.
```

Comment:
I'd sprinkle some bold to make them pop:

-  Your app or integration is **not accessible via free user sign-up** and **test credentials are not provided** to our review team for testing purposes.
- Your app **description is not complete** in the __Notes to reviewer__ section, and our review team **cannot test all the entrypoints**. Make sure you are clear about what your app integrates (full editor, modules, and/or quick actions) and where/how to invoke them.
- Your **app name should correspond with the name of your project** in Adobe Developer Console. This `appName` should also be consistent throughout your application. We use this property in our analytics checks so if there are any discrepancies, we will ask that you make that change before you are able to release the integration to your users.